snap/snapcraft.yaml

```diff
@@ -60,18 +59,11 @@ apps:
       - avahi-observe
       - browser-support
       - cups-control
-      - gsettings
       - opengl
-      - home
```

Comment:
I think we use the `home` plug for filesystem access so that you can drag+drop onto the composer window and we can resolve the file:// URLs we receive.

Reply:
(edit: I'm not 100% sure about this one, maybe we can try running a build without it and doing all sorts of file load / drag / mailto: link clicking and see if it all still works)

Reply:
This should be handled by XDG portals.